apps/main/src/components/PagePool/PoolDetails/ChartOhlcWrapper/index.tsx

```diff
@@ -14,12 +14,12 @@ import PoolActivity from '@/components/PagePool/PoolDetails/ChartOhlcWrapper/Poo
 import { getThreeHundredResultsAgo, subtractTimeUnit } from '@/ui/Chart/utils'
 import Box from '@/ui/Box'
 
-type Props = {
+type PoolInfoDataProps = {
   rChainId: ChainId
   pricesApiPoolData: PricesApiPool
 }
 
-const PoolInfoData = ({ rChainId, pricesApiPoolData }: Props) => {
+const PoolInfoData: React.FC<PoolInfoDataProps> = ({ rChainId, pricesApiPoolData }) => {
```

Comment:
It's not about defining children, it's about a convenient way to write the component type, including parameters and return value. I like it because it is easy to write and easy to read
https://dev.to/elhamnajeebullah/react-typescript-what-is-reactfc-and-why-should-i-use-it-4029